plugin/Gui/AdminPageApiSettings.php

```diff
@@ -617,7 +617,7 @@ private function addFormModelCache()
 	 */
 	private function addFormModelSeparatorFormatSettings(string $pageSlug)
 	{
-		$groupSlugPaging = 'onoffice-settings';
+		$groupSlugPaging = 'onoffice-thousand-separator';
```

Comment:
I think we should only change the GroupSlug in addFormModelSocialMetaData. If we change it in addFormModelSeparatorFormatSettings, users who have saved settings might need to save the SeparatorFormat settings again.